### PR TITLE
Use initlog() in more tests.

### PR DESCRIPTION
**Pull request by drwells**
Part of #4250. I was reminded of this when reviewing one of the PETSc patches - its pretty easy to automatically convert many of these with a python script.

### CODE REVIEW

File: tests/bits/step-10-high-order.cc
Comment (tamiko) [inline tests/bits/step-10-high-order.cc:183]: double `deallog <<`

File: tests/bits/step-10-high-order.cc
Comment (tamiko) [inline tests/bits/step-10-high-order.cc:184]: and you don't set the precision of the logfile any more (as you do for some other tests).

**Review by tamiko — CHANGES_REQUESTED** [review]
It looks like the conversion did not completely work for a small number of tests.

**Comment by drwells** [comment]
The tests passed locally for me but it's worth cleaning things up - both comments should now be fixed in all files.

**Review by masterleinad — APPROVED** [review]

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
@tamiko is still blocking the merge. I'll let him comment/pull the trigger.

**Review by tamiko — APPROVED** [review]

**Comment by tamiko** [comment]
@bangerth Feel free to simply dismiss such a change request from me as "resolved" in the future.

**Pull request merged**
